### DISCUSSION TITLE
Retain Handling flag on subscript a topic.

### DISCUSSION THREAD

**Discussion by cod1ingcoding** [emqx/MQTTX#1521] [General]
hi guys, how to understand Retain Handling flag?
![image](https://github.com/emqx/MQTTX/assets/1259507/f084d5d0-57e1-4d66-bf9e-010a3de2b04f)

**Comment by Red-Asuka** [comment]
@cod1ingcoding You can refer to this [article](https://www.emqx.com/en/blog/an-introduction-to-subscription-options-in-mqtt), which provides a detailed introduction to subscription options, including Retain Handling.
